Dockerfile

```diff
@@ -1,10 +1,36 @@
+### This is a generated file from Dockerfile.in ###
+#@follow_tag(openshift-golang-builder:1.14)
 FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7 AS builder
+
+ENV BUILD_VERSION=${CI_CONTAINER_VERSION}
+ENV OS_GIT_MAJOR=${CI_X_VERSION}
+ENV OS_GIT_MINOR=${CI_Y_VERSION}
+ENV OS_GIT_PATCH=${CI_Z_VERSION}
+ENV SOURCE_GIT_COMMIT=${CI_ELASTICSEARCH_PROXY_UPSTREAM_COMMIT}
+ENV SOURCE_GIT_URL=${CI_ELASTICSEARCH_PROXY_UPSTREAM_URL}
+ENV REMOTE_SOURCE=${REMOTE_SOURCE:-.}
+
+
+LABEL \
```

Comment:
@vimalk78  I moved this here to make local building better, but will it result in the final image not having these labels?  cc @sosiouxme  do you know?

Reply:
If the `LABEL` is here, i get this error in OSBS
```
FAILED: BuildError: Required LABELs haven't been found in Dockerfile: com.redhat.component (or BZComponent), name (or Name), version (or Version).
```

It builds fine if the `LABEL` is moved to end of file

Reply:
ill check it.  i think the labels have to probably be in the final image